packages/regl-worldview/src/utils/commandUtils.js

```diff
@@ -98,6 +98,14 @@ export const blend = {
   },
 };
 
+// regl default depth values that can be overridden by props
+export const depth = {
```

Comment:
I think there might've been discussion about this somewhere already but I can't find it now — did you see about whether you can make the whole `depth` object a function? Would be nicer for `props.depth` to mirror the type supported by regl rather than splitting it into `depthEnable`, `depthMask` etc.

Reply:
yup I wanted to do the whole 'depth object function' initially but it didn't seem to work, hence why I made this janky `depthEnable` api :(